ftest.sh

```diff
@@ -145,6 +145,18 @@ fi
 trap 'set +e; cleanup' EXIT
 
 CLUSH_ARGS=($CLUSH_ARGS)
+if ! clush "${CLUSH_ARGS[@]}" -B -l "${REMOTE_ACCT:-jenkins}" -R ssh -S \
+    -w "$(IFS=','; echo "${nodes[*]}")" "set -ex
+# set memlock to unlimited
+sudo bash -c \"set -ex
+if [ \\\"\\\$(ulimit -l)\\\" != \\\"unlimited\\\" ]; then
+    echo \\\"*  soft  memlock  unlimited\\\" >> /etc/security/limits.conf
+    echo \\\"root  soft  memlock  unlimited\\\" >> /etc/security/limits.conf
+    echo \\\"session required pam_limits.so\\\" > /etc/pam.d/common-session-noninteractive
+    echo \\\"session required pam_limits.so\\\" > /etc/pam.d/common-session
+    echo \\\"session required pam_limits.so\\\" >> /etc/pam.d/login
+fi\""; then
+fi
```

Comment:
(lint) Can't have empty then clauses (use 'true' as a no-op). [SC1048]
(lint) Expected "#". Fix any mentioned problems and try again. [SC1072]